packages/devtools_app/lib/src/screens/deep_link_validation/deep_link_list_view.dart

```diff
@@ -89,7 +89,19 @@ class _DeepLinkListViewMainPanel extends StatelessWidget {
         valueListenable: controller.allLinkDatasNotifier,
         builder: (context, linkDatas, _) {
           if (linkDatas == null) {
-            return const CenteredCircularProgressIndicator();
+            return Center(
```

Comment:
You should be able to delete this widget since you have `mainAxisAlignment: MainAxisAlignment.center,`. You may need to also add a centered `crossAxisAlignment` to the Column